charts/timescaledb-single/Chart.yaml

```diff
@@ -4,9 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.32.0
-icon: https://cdn.iconscout.com/icon/free/png-256/timescaledb-1958407-1651618.png
```

Comment:
No need to delete the `icon:` line here.

Reply:
yes, it was due to the rebased. Reverted